data/pi4_dashboard_data.json

```
@@ -1,4 +1,6 @@
 {
+  "generated_at": "2025-11-03T00:42:27.788914+00:00",
```

Biome (2.1.2)

[error] The key generated_at was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)